projects/plugins/jetpack/extensions/blocks/subscriptions/view.scss

```diff
@@ -61,16 +65,11 @@
 		}
 	}
 
-	&.wp-block-jetpack-subscriptions__show-subs {
-		padding-bottom: 32px;
-
-		.wp-block-jetpack-subscriptions__subscount,
-		.jetpack-subscribe-count p {
-			position: absolute;
-			bottom: 0;
-			right: 0;
-			margin: 0;
+	&.wp-block-jetpack-subscriptions__show-subs {		
+		.wp-block-jetpack-subscriptions__subscount {
+			margin: 8px 0;
 			font-size: 16px;
```

Comment:
Note: This was previously `padding-bottom: 32px`. So `8px + 16px + 8px` we get roughly the same `32px` as before.